Dockerfile.master.template

```diff
@@ -1,18 +1,10 @@
-FROM postgres:%%PG_MAJOR%%
+FROM postgres:%%PG_MAJOR%% as builder
```

Comment:
imho: Please add some Warning comment to the top of the master Dockerfile; 
At least an "experimental"  warning ..

~ like
```
# "experimental" ;  only for testing!
# multi-stage dockerfile;  minimal docker version >= 17.05
```

Reply:
Thank you comments.
I resolve all comments.